staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -193,3 +193,144 @@ type JobCondition struct {
 	// +optional
 	Message string `json:"message,omitempty" protobuf:"bytes,6,opt,name=message"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.8
+// +k8s:prerelease-lifecycle-gen:deprecated=1.22
```

Comment:
`k8s:prerelease-lifecycle-gen` annotations shouldn't apply to non-alpha/beta types. mind spawning an issue for @deads2k to catch this in CI validation?

Reply:
Yikes, missed it.